test/e2e/pivoting.go

```diff
@@ -83,6 +83,19 @@ func pivoting(ctx context.Context, inputGetter func() PivotingInput) {
 		fetchContainerLogs(&ironicContainers, input.ArtifactFolder, input.E2EConfig.GetVariable("CONTAINER_RUNTIME"))
 	}
 
+	By("Fetch target cluster kubeconfig for target cluster log collection")
+	kubeconfigPath := input.TargetCluster.GetKubeconfigPath()
+	os.Setenv("KUBE_CONFIG", kubeconfigPath)
```

Comment:
I would suggest using the standard `KUBECONFIG` and make sure to unset it after we are done. If we cannot use the standard variable, then maybe a name that explains better which cluster it is for. For example `KUBECONFIG_WORKLOAD` or similar.

Reply:
It will be handled in separate followup PR